tests/MaxMind/Test/MinFraud/Model/BillingAddressTest.php

```diff
@@ -20,7 +21,7 @@ public function testBillingAddress()
         ];
         $billing = new BillingAddress($array);
 
-        $this->assertSame(
+        $this->assertTrue(
```

Comment:
This change is incorrect as you are no longer testing `$billing->isPostalInCity`. You are testing the `$array` value. The same issue exists with the other boolean tests.